src/poker/game/ActionHandler.java

```diff
@@ -69,6 +75,24 @@ public void handleNextHand() {
 //        }
     }
 
+    public  void handlePrintEquity(Integer i){
+        HandsAndBoard handsAndBoard = getHandsAndBoard();
+       List<Card> board = handsAndBoard.getBoard();
+       List<Card> hand1 = handsAndBoard.getHand1();
+       List<Card> hand2 = handsAndBoard.getHand2();
+        if (i == 0){
```

Comment:
Byniu skoro mamy Enuma PREFLOP FLOP TURN I RIVER to uzyj go a nie jakims gowno "i" mi tutaj robisz ify